android/src/ti/iap/models/PurchaseModel.kt

```diff
@@ -11,7 +11,7 @@ class PurchaseModel(val purchase: Purchase) {
             val modelDict = KrollDict()
 
             modelDict[PurchaseModelKeys.PURCHASE_TOKEN] = purchase.purchaseToken // primary-key and globally unique, can be used in database records safely
-            modelDict[PurchaseModelKeys.PRODUCT_ID] = purchase.sku
+            modelDict[PurchaseModelKeys.PRODUCT_ID] = purchase.products.toString()
```

Comment:
This is an array, will will result in an invalid value, e.g. `[my.product]` instead of `my.product`. Since it's actively used to start and log a purchase, this will cause issues.